qa/tasks/cephfs/test_volume_client.py

```diff
@@ -53,16 +55,8 @@ def _sudo_write_file(self, remote, path, data):
 
         Both perms and owner are passed directly to chmod.
         """
-        remote.run(
-            args=[
-                'sudo',
-                'python',
-                '-c',
-                'import shutil, sys; shutil.copyfileobj(sys.stdin, file(sys.argv[1], "wb"))',
-                path,
-            ],
-            stdin=data,
-        )
+        remote.run(args=['sudo', 'python3', '-c',
```

Comment:
here also: python3 unconditionally?

Reply:
This can be and needs to be changed. All I did is run `s/python2/python3`.

Reply:
Done.